starknet-providers/src/jsonrpc/mod.rs

```diff
@@ -224,11 +226,15 @@ where
     /// Get the contract class definition associated with the given hash
```

Comment:
The description for this method has changed to `Get the contract class definition in the given block associated with the given hash` in the specs. Please reflect it here in the docs. Please also check for other desc changes.